libs/sr-formula/src/data/relic/index.ts

```diff
@@ -1,4 +1,18 @@
+import { allRelicSetKeys } from '@genshin-optimizer/sr-consts'
+import { allStats } from '@genshin-optimizer/sr-stats'
 import type { TagMapNodeEntries } from '../util'
+import FirmamentFrontlineGlamoth from './FirmamentFrontlineGlamoth'
+import PrisonerInDeepConfinement from './PrisonerInDeepConfinement'
+import { entriesForRelic } from './util'
 
-const data: TagMapNodeEntries = []
+const data: TagMapNodeEntries = allRelicSetKeys.flatMap((key) => {
+  switch (key) {
+    case 'FirmamentFrontlineGlamoth':
```

Comment:
nit: same maintainability concern as before

Reply:
Same, will nuke in next pr